cargo-concordium/src/main.rs

```diff
@@ -684,9 +692,15 @@ pub fn main() -> anyhow::Result<()> {
         Command::Test {
             seed,
             build_options,
+            test_threads,
             only_unit_tests,
             test_targets,
         } => {
+            if let Some(test_threads) = test_threads {
```

Comment:
In the previous commit, this if statement was missing. This still worked as passing the default `0` to rayon is the same as not calling this code at all. However, since this code could fail if `.build_global()` in the future is called twice at runtime, I would rather have it only fail if a flag is specifically passed.